incubator/podgrab/1.0.0/Chart.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: podgrab
+version: 1.0.0
+upstream_version: 1.0.0
+appVersion: "latest"
+description: Podcast downloader and player
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/incubator/podgrab
+icon: https://github.com/warllo54/apps/blob/podgrab/incubator/podgrab/icon.png
```

Nope.

---

Any reason the code below wouldn't work or would have a suggestion on how to do it?

```suggestion
https://github.com/truecharts/apps/tree/master/incubator/podgrab/1.0.0/icon.png
```

---

Better to use a link from the official repo, if there is one.
Alternative check this -> https://github.com/truecharts/apps/blob/master/stable/collabora-online/item.yaml

---

Also:
We link to icons on the website for a while now, so putting icons into the app itself is not allowed anymore. See other example apps.

---

Unfortunately, the official repo doesn't have an icon of any kind. Does `https://github.com/truecharts/apps/tree/master/docs/_static/img` point to `https://truecharts.org/_static/img/` ?

I guess I'm not sure where the website is sourcing it's images from.

---

> @stavros-k Unfortunately, the official repo doesn't have an icon of any kind. Does `https://github.com/truecharts/apps/tree/master/docs/_static/img` point to `https://truecharts.org/_static/img/` ?
 
 Correct